Projects/Features/SearchFeature/Sources/ViewControllers/SongSearchResultViewController.swift

```diff
@@ -58,19 +63,39 @@ final class SongSearchResultViewController: BaseReactorViewController<SongSearch
             .map { _ in SongSearchResultReactor.Action.askLoadMore }
             .bind(to: reactor.action)
             .disposed(by: disposeBag)
+
+        headerView.rx.tapSortButton
+            .withLatestFrom(sharedState.map(\.sortType))
+            .bind(with: self) { owner, sortType in
+                guard let vc = owner.searchSortOptionComponent.makeView(sortType) as? SearchSortOptionViewController
+                else {
+                    return
+                }
+
+                vc.delegate = owner
+
+                owner.showBottomSheet(
+                    content: vc,
+                    size: .fixed(240 + SAFEAREA_BOTTOM_HEIGHT())
```

Comment:
<img width="715" alt="image" src="https://github.com/wakmusic/wakmusic-iOS/assets/60254939/e4aaa53a-9c38-4734-9e36-3516e6e22b5d">

```suggestion
                    size: .fixed(248 + SAFEAREA_BOTTOM_HEIGHT())
```
높이 248이 맞나요?

Reply:
상, 하단 마진(20+20) + 항목갯수(하나당 52)라고하면 인기순이 없으니 (20+52+52+52+20) = 196이 맞음.
하지만, 바텀시트의 경우 콘텐츠가 적어서 너무 바닥에 붙어있으면 손가락이 움직여야 하는 범위가 넓어지므로 하단 마진을 일부러 넉넉하게 주는 경우가 있긴 했던걸로 기억함.